### ISSUE TITLE
Fails to load SDK test app in sandbox

### ISSUE THREAD

**Issue by fernandomg** [monerium/sdk#31]
Hello,

I was following the dev docs, in https://monerium.dev/docs/getting-started/create-app

After clicking the link "Click here to view an example of an app onboarding", I was prompted with what's on the image.

<img width="480" alt="image" src="https://user-images.githubusercontent.com/3315606/230092541-f186f7fe-059b-46d2-88b7-fefbf710b9a7.png">

After clicking "Accept" tries to reach https://sandbox.monerium.dev/api/iam/auth which responds with the following HTTP 302 Location: `http://localhost:5173/integration?code=<code>&state=`

Hope that information is enough for you. I'm happy to help with whatever other feedback you need.

Thank you!

**Comment by einaralex** [comment]
Hey! Sorry for the late reply. 

Thank you, yeah that is confusing; this is the URL that I had specified in the `Redirect URL`  when I created the app.

I've changed it from `http:localhost:5173/integration` to `https://www.example.com/your-application`, that should be more clear?

<img width="472" alt="image" src="https://user-images.githubusercontent.com/2564144/233062768-8c2e052f-9675-4318-9dc0-62ce1e760239.png">
